packages/main/src/ListItem.ts

```diff
@@ -165,6 +166,16 @@ abstract class ListItem extends ListItemBase {
 	@property()
 	title!: string;
 
+	/**
+	 * Defines the highlight state of the list items.
+	 * Available options are: `"None"` (by default), `"Success"`, `"Warning"`, `"Information"` and `"Error"`.
+	 * @default "None"
+	 * @public
+	 * @since 1.24
+	 */
+	@property({ type: ValueState, defaultValue: ValueState.None })
+	highlight!: `${ValueState}`;
```

Comment:
This is inconsistent:
highlight!: `${ValueState}`;

ValueState is strongly linked with input components and imply slighly different semantics

To be on the safe side, please create a new enum
HighlightTypes with the same values
"None" | "Success" | "Warning" | "Error" | "Info

Later we can extract those on more generic enum like States or similar that both HighlightTypes and ValueState can reuse.